x-pack/plugins/rule_registry/server/routes/get_alert_summary.ts

```diff
@@ -57,10 +57,8 @@ export const getAlertSummaryRoute = (router: IRouter<RacRequestHandlerContext>)
           throw Boom.badRequest('gte and/or lte are not following the UTC format');
         }
 
-        if (fixedInterval && fixedInterval?.match(/^\d{1,2}['m','h','d','w']$/) == null) {
```

Comment:
Fixed interval does not have `w` ([link](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-datehistogram-aggregation.html#fixed_intervals))